src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier loteria = new ColorSupplier();
```

Comment:
The variable name 'loteria' should be more descriptive and follow Java naming conventions. Consider renaming it to 'colorSupplier' to clearly reflect its purpose.